docs_source_files/content/support_packages/mouse_and_keyboard_actions_in_detail.de.md

```diff
@@ -233,7 +263,17 @@ public class moveToElement {
 }
   {{< / code-panel >}}
   {{< code-panel language="python" >}}
-# Please raise a PR
+from selenium import webdriver
+driver = webdriver.Chrome()
+
+# Navigate to url
+driver.get("http://www.google.com")
+
+# Store 'google search' button web element
+searchBtn = driver.find_element_by_link_text("Gmail")
```

Comment:
can you please change variable name to 'gmailLink' and also can you update the comment.